react/components/QuoteDetails/SaveButtons.tsx

```diff
@@ -30,7 +30,7 @@ const SaveButtons = ({
               updatingQuoteState
             }
           >
-            <FormattedMessage id="store/b2b-quotes.create.button.save-for-later" />
+            fdsdf
```

Comment:
I thinks this is some test code that ended up on the PR

Reply:
resolved